### DISCUSSION TITLE
doris查询

### DISCUSSION THREAD

**Discussion by 2298877046** [apache/doris#42277] [A - General / Q&A]
在java里执行一段原生的doris查询比如：
public List getTalentInterestPortrait(DataContentListQuery query) {

    List<TalentPortraitVo> talentGenderPortraitList =  dorisTalentMapper.getTalentInterestPortrait(query);

}
我想要获取 List talentGenderPortraitList = dorisTalentMapper.getTalentInterestPortrait(query);这段查询他的数据库线程标识，就是使用命令show processlist对应的id，要怎么获取，求求大佬解答

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。里面有专人值班
